frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/mirroring-card/mirroring-card.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import * as classNames from 'classnames';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import {
+  DashboardItemProps,
+  withDashboardResources,
+} from '@console/internal/components/dashboard/with-dashboard-resources';
+import { K8sResourceKind } from '@console/internal/module/k8s/index';
+
+import { StorageClusterKind } from '../../../../types';
+import { ocsResource } from '../../../../resources';
+import '../../common/storage-efficiency/storage-efficiency-card.scss';
+
+export const MirroringItemBody: React.FC<MirroringItemBodyProps> = React.memo(
```

Comment:
ToDo:
  I am creating MirroringCardBody and MirroringCardItem components in my dashboard PR: https://github.com/openshift/console/pull/9423/files#diff-aa479e95de24a9a068e30fa9ab25f8e33574f37c8ee80160eaa3a7d5b627b0d8R5
  
 We will investigate more and try to merge this MirroringItemBody into those.
 
 For now, we are keeping component this separate.